README.md

```diff
@@ -14,14 +14,19 @@ prerequisite for developing FastBoot apps.
 
 ### Command Line
 
-You can run the FastBoot server from the command line:
+You can run the FastBoot server from the command line by providing the HTML file as well as
+your `vendor.js` and `app.js` that have been built with `ember fastboot:build`. You can also pass
+addition `js` files. Each file will be loaded in the order it appears in the command line.
 
-```
+```sh
 $ ember-fastboot \
-    --port 80
-    --app-file path/to/app-file.js
-    --vendor-file path/to/vendor-file.js
-    --html-file path/to/index.html
+    --host 0.0.0.0 \
+    --port 80 \
+    --html-file path/to/index.html \
+    path/to/vendor-file.js \
+    path/to/app-file.js \
+    path/to/other-file-1.js \
+    path/to/other-file-2.js
```

Comment:
I got the idea from when you mentioned you're working on testing stuff and thinking about how I would go about executing tests using the fastboot command line.. For example if you had the fastboot specific test code and it needs to run after all your JS or before it all - or in-between some groups of files.. 

```
ember-fastboot \
    --host 0.0.0.0 \
    --port 80 \
    --html-file dist/tests/index.html \
    path/to/vendor-file.js \
    path/to/app-file.js \
    path/to/fastboot-test-support.js
```

Also the thought was there that the ability to split the files into more than just two (vendor/app) could allow for more flexible or smarter live code reloading (whenever/if ever that comes) as well as potentially reusing the files that are compiled with regular `ember build` command - e.g. if whatever is added/changed in your app by using the command `ember fastboot:build` could instead be split out into dedicated files - then there wouldn't be a need to recreated vendor/app

Also partly influenced by not wanting to type --app-file and --vendor-file